docs/index.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   conjur-config-cluster-prep:
   - apiVersion: v2
-    created: "2021-09-30T01:19:08.23103-07:00"
+    created: "2021-10-31T12:32:34.199223-04:00"
```

Comment:
Why did all of these other dates get updated? That doesn't seem right, but I'm guessing it just happens based on the fact that all of them are the same.

Reply:
This is something that the `helm repo index ...` command does. I don't know if there's a flag we can add to change this behavior. Here's the command we use to generate these changes (from `reindex.sh`):
```
helm repo index docs --url https://cyberark.github.io/helm-charts
```